services/value-store/projects/src/mappings/project.js

```diff
@@ -0,0 +1,50 @@
+module.exports = {
+  project: {
+    properties: {
+      budget: {
+        properties: {
+          funding_area: {
+            type: 'keyword',
+          },
+        },
+      },
+      computed_key: {
+        type: 'keyword',
+      },
+      coordinators: {
+        properties: {
+          country: {
+            type: 'keyword',
+          },
+        },
+      },
+      ec_priorities: {
+        type: 'keyword',
+      },
+      producer_id: {
+        type: 'keyword',
+      },
+      programme_name: {
+        type: 'keyword',
+      },
+      project_locations: {
+        properties: {
+          location: {
+            type: 'geo_point',
+            ignore_malformed: true,
```

Comment:
Is there a specific reason for `ignore_malformed: true`?

Reply:
This one I added before we make latitude and longitude 0 if not available. If a value is not 0, but `null` as in most defaults in our case, indexing fails and stops the process.